python-sdk/src/astro/files/base.py

```diff
@@ -250,18 +257,18 @@ def resolve_file_path_pattern(
     :param normalize_config: parameters in dict format of pandas json_normalize() function
     """
     location = create_file_location(path_pattern, conn_id)
-
-    files = [
-        File(
+    files = []
+    for path in location.paths:
+        if path.endswith("/"):
+            continue
```

Comment:
Should we add a unit test to cover this?

Reply:
Changed condition to get better coverage.